test/unit/org/apache/cassandra/db/RecoveryManagerTest.java

```diff
@@ -250,28 +249,84 @@ public void testRecoverCounter() throws IOException
     public void testRecoverPIT() throws Exception
     {
         CommitLog.instance.resetUnsafe(true);
+        long rpiTimeInit = CommitLog.instance.archiver.restorePointInTimeInMicros;
```

Comment:
Whenever I see `rpi` my brain parses it as Raspberry Pi 🤓 sorry!
What calling it `originalPIT` ? Later we'll be doing `.setRestorePointInTime(originalPIT);`